src/Components/Facility/FacilityHome.tsx

```diff
@@ -774,6 +774,14 @@ export const FacilityHome = (props: any) => {
               const doctorRes = await dispatch(listDoctor({}, { facilityId }));
               if (doctorRes && doctorRes.data) {
                 setDoctorData(doctorRes.data.results);
+                // update total doctors count
+                let totalCount = 0;
+                doctorRes.data.results.map((doctor: DoctorModal) => {
+                  if (doctor.count) {
+                    totalCount += doctor.count;
+                  }
+                });
+                setTotalDoctors(totalCount);
```

Comment:
@Pranshu1902 use `reduce` instead of `map`

Reply:
https://stackoverflow.com/a/16751601/7887936

Reply:
Done